substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -171,153 +170,32 @@ struct DefaultCmd {
 /// can be updated with the runtime provided in the command line.
 #[derive(Parser, Debug, Clone)]
 pub struct EditCmd {
-	/// Chain spec to be edited
+	/// Chain spec to be edited.
 	#[arg(long, short)]
```

Comment:
As @skunert said, this should be some required argument.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/2174#discussion_r1399779900

Reply:
done in: ed84b6b